src/webhint-theme/layout/partials/scan-queue.ejs

```diff
@@ -18,14 +18,14 @@
                         <p>You can check back later using the permalink below. This is your unique scan result URL so you can return to your results page at any time.</p>
                         <div class="scan-queue__permalink">
                             <p class="uppercase-text">Permalink</p>
-                            <button class="permalink-copy">
+                            <button type="button" class="permalink-copy">
                                 <span class='permalink-content'><%= result.permalink %></span> <img src="/images/scan/permalink-icon.svg" />
```

Comment:
You didn't make this update but one of the errors I am getting via the extension is that this img needs an alt tag. Can you add an alt="copy permalink" ?

Reply:
Done